hydra_base/lib/storage/readers.py

```diff
@@ -130,13 +130,21 @@ def get_series_by_column_names(self, column_names, start=None, end=None):
         return column_series
 
     def find_index_axis_index(self):
+        # If index is a Pandas type, an "index_class" attr
+        # will be present to indicate the index
         for ent in self.group:
             try:
                 index_class = self.group[ent].attrs["index_class"]
                 return ent
             except KeyError:
                 continue
 
+        # No item has a designated index, assume axis1 is index,
+        # as per non-Datetime/Period Pandas index
```

Comment:
Is this a guarantee based on looking at the way pandas encodes the data?

Reply:
Yes, the section above filters out timeseries indexes leaving only cases where the index is a plain type, and in these cases `axis0` contains the column names and `axis1` contains the index.  As with the whole process, this relies on Pandas' internal use of Pytables to encode a DataFrame in an HDF file, which isn't ideal but that's the input format we have.